packages/edit-site/src/components/global-styles/screen-typeset.js

```diff
@@ -0,0 +1,43 @@
+/**
+ * WordPress dependencies
+ */
+import { __ } from '@wordpress/i18n';
+import { useSelect } from '@wordpress/data';
+import { store as editorStore } from '@wordpress/editor';
+import { __experimentalVStack as VStack } from '@wordpress/components';
+
+/**
+ * Internal dependencies
+ */
+import TypographyVariations from './variations/variations-typography';
+import ScreenHeader from './header';
+import FontFamilies from './font-families';
+
+function ScreenTypeset() {
+	const fontLibraryEnabled = useSelect(
+		( select ) =>
+			select( editorStore ).getEditorSettings().fontLibraryEnabled,
+		[]
+	);
+
+	return (
+		<>
+			<ScreenHeader
+				title={ __( 'Typesets' ) }
+				description={ __(
+					'Fonts and typographic styling applied across the site.'
+				) }
+			/>
+			<div className="edit-site-global-styles-screen">
+				<VStack spacing={ 7 }>
+					<TypographyVariations />
+
+					{ ! window.__experimentalDisableFontLibrary &&
```

Comment:
Where is `__experimentalDisableFontLibrary` defined? I can't see it referenced anywhere else.

Reply:
Looks like the experiment was removed but this was left in. I'll remove it in a different PR to keep concerns separated.

Reply:
Done here:  https://github.com/WordPress/gutenberg/pull/63890